docs/docs/configuration/queryable-projections.mdx

```diff
@@ -76,3 +74,48 @@ public static partial class CarMapper
     private static partial CarDto Map(Car car);
 }
 ```
+
+## User-implemented mapping methods
+
+Mapperly tries to inline user-implemented mapping methods.
+For this to work, user-implemented mapping methods need satisfy certain limitations:
+
+- Only expression-bodied methods can be inlined.
+- The body needs to follow the expression tree [limitations](https://learn.microsoft.com/en-us/dotnet/csharp/advanced-topics/expression-trees/#limitations).
+- Nested MethodGroups cannot be inlined.
+
+```csharp
+[Mapper]
+public static partial class CarMapper
+{
+    public static partial IQueryable<CarDto> ProjectToDto(this IQueryable<Car> q);
+
+    // highlight-start
+    private static string MapCarBrandName(CarBrand brand)
+      => band.Name;
+    // highlight-end
+}
+```
+
+If Mapperly is unable to inline a user-implemented method, `RMG068` is reported.
+Non-inlined method invocations can cause the query to be materialized and more data than necessary could be loaded.
+
+Ordering and aggregating collections can also be implemented with user-implemented mapping methods:
+
+```csharp
+[Mapper]
+public static partial class CarMapper
+{
+    public static partial IQueryable<CarDto> ProjectToDto(this IQueryable<Car> q);
+
+    private static partial CarModelDto MapCarModel(CarModel model);
+
+    // highlight-start
+    private static ICollection<CarModelDto> MapOrderedCarBrandName(ICollection<CarModel> models)
+      => models.OrderBy(x => x.Name).Select(x => MapCarModel(x)).ToList(); // note: do not use the method group
+    // highlight-end
+  }
+```
+
+It is important that the types in the user-implemented mapping method match the types of the objects to be mapped exactly.
+Otherwise, Mapperly may not resolve the user-implemented mapping methods.
```

Comment:
Nitpick: I think "cannot" is more appropriate here instead of "may not"

Reply:
Improved